sherpa/astro/ui/utils.py

```diff
@@ -13248,6 +13248,10 @@ def sample_flux(self, modelcomponent=None, lo=None, hi=None, id=None,
         values used, together with the median, upper, and lower
         quantiles of the flux distribution.
 
+        .. versionchanged:: 4.13.1
```

Comment:
I noticed that the `Returns` section in the docsstring has an unusual formatting. Should that be modified?

Reply:
What is the problem with it? It at least renders "okay" at https://sherpa--1076.org.readthedocs.build/en/1076/ui/api/sherpa.astro.ui.sample_flux.html but I am likely missing something obvious